src/openrct2/peep/Peep.cpp

```diff
@@ -2047,28 +2048,29 @@ bool Peep::SetName(const std::string_view& value)
  */
 void peep_thought_set_format_args(const rct_peep_thought* thought)
 {
-    set_format_arg(0, rct_string_id, PeepThoughts[thought->type]);
+    auto ft = Formatter::Common();
+    ft.Add<rct_string_id>(PeepThoughts[thought->type]);
```

Comment:
Why not joining these in the same line, like you did on `Paint.Path.cpp `?

Reply:
I start a new formatter when the "set_format_arg" is writing to the position 0, and the next set_format_args write incrementally (2, 4...).

Reply:
You can see that in Paint.Path.cpp it writes to 0, then 4, then starts again writing to 0.

Reply:
I think it could've been collapsed, but this is not a huge deal, so let's move on